### ISSUE TITLE
Making a proper tutorial about how to use this plugin.

### ISSUE THREAD

**Issue by NycuR0** [Ad5001/UHC#10]
@Ad5001 Can you speak me how work?

For me ,not work plugin...

**Comment by Ad5001** [comment]
What do you mean?

**Comment by Shock95** [comment]
@NycuR0 Please explain your problem, we cannot answer your issue unless you do so.

**Comment by ghost** [comment]
@XShockinFireX @Ad5001  You work this plugin?

How i can setup world and other?

**Comment by TiesGamesNL** [comment]
how to do the sign????

**Comment by ghost** [comment]
Yes, How create arena ,how enter.. commands ,how setup plugin? :/

**Comment by TiesGamesNL** [comment]
idk i tried it too {dont know how to do the signs}

**Comment by Ad5001** [comment]
I see.
Copy the example config from the readme, change the name "uhc" by your world name and then you will have created an UHC world.

**Comment by TiesGamesNL** [comment]
how to create a sign

**Comment by Ad5001** [comment]
I don't think I added a way to do "teleport signs" did I? Made this a long time ago.

**Comment by Ad5001** [comment]
But you could use an another plugin such as TapToDo to make a sign that teleports the player in an another world.

**Comment by TiesGamesNL** [comment]
thats resetting everytime

**Comment by Ad5001** [comment]
?

**Comment by TiesGamesNL** [comment]
taptodo doesnt work for me

**Comment by Ad5001** [comment]
Oh ok.

**Comment by ghost** [comment]
@Ad5001 Ok using TapToDo. But what players need to start and how use ? You can create one video with setup?

**Comment by Ad5001** [comment]
The game automaticly starts when 75% of the maxplayers has joined the area. But you can manualy start it with /uhc start

**Comment by ghost** [comment]
@Ad5001 Ok ,but need Sign arena like SW and other minigames.

**Comment by ghost** [comment]
With TapToDo , ok. But you can add more messages in chat or other for watch if is correct setted or other?

Or one video?

**Comment by Ad5001** [comment]
I will see about this...
